src/uploaders.rs

```diff
@@ -61,11 +65,14 @@ impl Uploader {
     /// It returns `None` if the current `Uploader` is `NoOp`.
     pub fn readme_location(&self, crate_name: &str, version: &str) -> Option<String> {
         match *self {
-            Uploader::S3 { ref bucket, .. } => Some(format!(
-                "https://{}/{}",
-                bucket.host(),
-                Uploader::readme_path(crate_name, version)
-            )),
+            Uploader::S3 { ref bucket, .. } => {
```

Comment:
Looks like this line needs to be changed to `Uploader::S3 { ref bucket, ref cdn, .. } => {`